panel/models/column.ts

```diff
@@ -62,7 +62,7 @@ export class ColumnView extends BkColumnView {
 
   override render(): void {
     super.render()
-    this.scroll_down_button_el = DOM.createElement("div", {class: "scroll-button"})
+    this.scroll_down_button_el = DOM.create_element("div", {class: "scroll-button"})
```

Comment:
`createElement()` wasn't meant to be exposed. I would suggest using `div(...)` for non variadic cases.